server/clearbit.js

```diff
@@ -246,24 +255,24 @@ export default class Clearbit {
     const domain = getDomain(user);
 
     if (!domain) {
-      this.debug("Skip Prospect - unknown domain");
+      this.hull.asUser(_.pick(user, ["id", "external_id", "email"])).logger.info("outgoing.user.skip", { reason: "No domain", action: "prospector" });
```

Comment:
let's factor the whole `this.hull.asUser(_.pick(user, ["id", "external_id", "email"])).logger.info("outgoing.user.skip",...` thing in a method that logs the reason

what about somehing like : 

`this.skip(user, "No domain", action: "prospector")`